### PR TITLE
test for checking deletion of orphaned langs

### LINKED ISSUE #344: Languages/Categories should delete themselves if they no longer exist
Suppose we have a language called `CSS/Bootstrap` and then someone realized that that's two languages and then changes the languages for that resource to be `["CSS","Bootstrap"]` instead. What happens to the language `CSS/Bootstrap`? Well, it gets orphaned. It exists, but there are not resources that have that language.

I think that whenever a resource is updated and the `languages` attribute is one of the changes, we should check that the old languages still exist on some resource. If they do not, then we should delete the language.

Same thing with categories. If a category gets removed from the last resource with that category, then the category should be deleted as well.

### PR DESCRIPTION
**Pull request by king-11**
random name for language use to prevent collision
test for #362 to close #344

### CODE REVIEW

File: tests/unit/test_routes/test_resource_update.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from tests.unit.test_response_util import client
```
Comment (aaron-suarez) [inline tests/unit/test_routes/test_resource_update.py:1]: Is this line necessary? I think later on in the file `client` is going to refer to `module_client` which is passed in by pytest, so this is actually an unused import, right?

Comment (king-11) [reply tests/unit/test_routes/test_resource_update.py:1]: yeah it is indeed unused

**Review by king-11 — COMMENTED** [review]
made the requested changes

**Comment by aaron-suarez** [comment]
Closed via #362

**Pull request closed without merging**